vcd/config_test.go

```diff
@@ -67,6 +67,8 @@ type TestConfig struct {
 		ApiToken                string `json:"api_token,omitempty"`
 		ApiTokenFile            string `json:"api_token_file,omitempty"`
 		ServiceAccountTokenFile string `json:"service_account_token_file,omitempty"`
+		VcdVersion              string `json:"vcdVersion,omitempty"`
+		ApiVersion              string `json:"apiVersion,omitempty"`
```

Comment:
I would like to ask for comments above these two fields. It would be good if they answered how did we live without them until now and why do we need to specify these versions explicitly now :)

Please also extend the PR description with such explanation.

Reply:
Out of curiosity, how much time does it save?

BTW, might there be a scenario where you can't fill these in? Like you just point to some automatically generated CD instance which version you don't know up-front.

Reply:
It saves development time, and reduces the risk of excluding the test from binary runs.
Sometimes, we can't use this trick, as we need the temporary connection for more than checking the version.
This PR removes about half of the currently used temporary connections.